### ISSUE TITLE
Linting error when using `test.extend`

### ISSUE THREAD

**Issue by mickremedi** [veritem/eslint-plugin-vitest#281]
When running a simplified version of the `test.extend` demo from the vitest docs:

```ts
import { test } from "vitest";

export const myTest = test.extend({
  archive: []
})
```

I get the following linting errors:
```
error  Use expect in test body      vitest/expect-expect
error  Test title must be a string  vitest/valid-title
```

Link to Docs: https://vitest.dev/guide/test-context.html#extend-test-context
eslint-plugin-vitest: 0.3.8
vitest: 0.34.6

**Comment by veritem** [comment]
this is a new api that's not currently implemented

**Comment by luixo** [comment]
This is still an error in 1.0.4

**Comment by veritem** [comment]
those two rules are among recommended rules you can use all rules setting instead of recommended like this

```js
{
  "extends": ["plugin:vitest/all"]
}
``` 

or turn them off

**Comment by luixo** [comment]
This doesn't resolve the problem - it is a false positive in the rule and clearly a bug.

**Comment by veritem** [comment]
It'll be fixed in the next release

**Comment by csantos1113** [comment]
I'm still seeing these 3 errors, even on the latest version of this plugin.

<img width="693" alt="Screenshot 2024-04-04 at 11 47 33 AM" src="https://github.com/veritem/eslint-plugin-vitest/assets/9648559/e4d1e062-bbfa-4dcc-9661-73da55d6a00d">

<img width="555" alt="Screenshot 2024-04-04 at 11 47 37 AM" src="https://github.com/veritem/eslint-plugin-vitest/assets/9648559/77b37bf6-79bd-4974-b660-99556e59dbe7">

----

Could you pls link to which version includes the fix? thanks

**Comment by luixo** [comment]
@veritem Still reproduces on v0.5.4
